assets/templates/big-data-input.html

```diff
@@ -0,0 +1,467 @@
+            <h2>Big data | Custom field | Emojis | Consejos Subject</h2>
+          </div>
+        </div>
+        <span class="arrow"></span>
+      </div>
+    </header>
+    <section class="dp-container">
+      <div class="dp-rowflex">
+        <div class="col-lg-8 col-md-12 m-b-24">
+          <div class="dp-wrapper-form-plans">
+            <form class="dp-forms-plans">
+              <fieldset>
+                <legend></legend>
+                <ul class="field-group">
+                  <!-- utilizar la clase dp-validate en el li para mostrar mensajes de validacion -->
+                  <li class="field-item">
+                    <label for="email">Lorem ipsum:</label>
+                    <input
+                      type="email"
+                      name="email"
+                      id="email"
+                      placeholder="Lorem ipsum"
+                    />
+                    <div class="wrapper-errors dp-message dp-error-form">
+                      <p>¡Ouch! Este campo está vacío</p>
+                    </div>
+                  </li>
+                  <li class="field-item">
+                    <ul class="field-group">
+                      <!-- utilizar la clase dp-validate en el li para mostrar mensajes de validacion -->
+                      <li class="field-item field-item--50">
+                        <label for="name">Lorem ipsum:</label>
+                        <input
+                          type="text"
+                          name="name"
+                          id="name"
+                          placeholder="Lorem ipsum"
+                        />
+                        <div class="wrapper-errors dp-message dp-error-form">
+                          <p>¡Ouch! Este campo está vacío</p>
+                        </div>
+                      </li>
+                      <!-- utilizar la clase dp-validate en el li para mostrar mensajes de validacion -->
+                      <li class="field-item field-item--50">
+                        <div class="dp-wrapper-bd">
+                          <label for="lastname">*Asunto:</label>
+                          <input
+                            type="text"
+                            name="lastname"
+                            id="lastname"
+                            placeholder=""
+                          />
+
+                          <div class="dp-button-group">
+                            <div class="dp-bd-box">
+                              <button
+                                type="button"
+                                class="dp-button button-medium dp-button-bd"
+                                aria-controls="dp-options-cf"
+                              >
+                                <i class="ms-icon icon-brackets"></i>
+                              </button>
+                              <div class="dp-content-bd">
+                                <ul class="dp-list-cfield" id="dp-options-cf">
```

Comment:
To try an use the js in this library for tabulation you can try:
```suggestion
                                <ul class="dp-list-cfield" id="dp-options-cf" aria-labelledby="cf-button">
```
And then in the button above
```
<button
                                type="button"
                                class="dp-button button-medium dp-button-bd"
                                aria-controls="dp-options-cf"
                                id="cf-button"
                              >
```
Then in javascript index.html try initializing this button:
```
var menubutton = new Menubutton(document.getElementById('cf-button'));
    menubutton.init();
```
Then try and see if tabulation works.